app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -120,18 +104,81 @@ public void onProviderDisabled(String provider) {
     private FloatingActionButton mapCenterButton;
     //GPS attributes
     private LocationManager gpsmanager; //LocationManager instance to check gps activity
-
-
     // Search components
     private SearchView search;
     private POISearchAdapter poiSearchAdapter;
     private ExpandableListView searchList;
     private Dialog searchDialog;
-
     private POI location;
     private POI destination;
     private SearchState searchState;
 
+    /**
```

Comment:
yes I noticed. It was after merging with master. I will merge manually when team approves the fix.